src/web/app/components/adding-question-panel/adding-question-panel.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="card-body background-color-light-blue text-center">
+  <div ngbDropdown class="d-inline-block">
+    <button id="btn-new-question" class="btn btn-primary" ngbDropdownToggle>Add New Question</button>
+    <div id="new-question-dropdown" ngbDropdownMenu>
+      <button class="dropdown-item" (click)="templateQuestionModalHandler()">Choose from template questions</button>
+      <div class="dropdown-divider"></div>
+      <ng-container *ngFor="let questionType of feedbackQuestionTypes | enumToArray">
```

Comment:
Why does `feedbackQuestionTypes` need to be a variable? Available question types are no different regardless of whether it is help page or actual page.

Reply:
fixed in 850dcc8